src/main/scala/beam/agentsim/events/ParkingUtilityEvent.scala

```diff
@@ -0,0 +1,124 @@
+package beam.agentsim.events
+
+import java.util
+
+import beam.agentsim.agents.vehicles.{BeamVehicle, BeamVehicleType}
+import beam.agentsim.infrastructure.charging.ChargingPointType
+import beam.agentsim.infrastructure.parking.{ParkingMNL, ParkingType}
+import beam.agentsim.infrastructure.parking.ParkingZoneSearch.ParkingZoneSearchStats
+import org.matsim.api.core.v01.events.Event
+
+case class ParkingUtilityEvent(
+  driverId: String,
+  beamVehicle: Option[BeamVehicle],
+  activityType: String,
+  parkingDuration: Double,
+  agentVoT: Double,
+  parkingZoneSearchStats: ParkingZoneSearchStats,
+  selectedStallPrice: Double,
+  selectedStallParkingType: ParkingType,
+  selectedStallChargingPointType: Option[ChargingPointType],
+  tick: Double = -1 // todo maybe we need the currentTick? If yes we have to get it into ZonalParkingManager
```

Comment:
yes. i do think we need the current tick here in order to make sense of what we observe, for example, to compare against trends in ParkEvents and RefuelSessionEvents (estimated vs experienced parking utility).

we can probably send it via the ParkingInquiry from the various spots where we send those for PEVs and ride hail.

Reply:
Agree. I tried to implement it exactly like this, but parkingInquiry is send from several places and not everywhere it is send from is the current tick available. 
=> it would need some bigger refactoring in several places

But I'm going to adapt theses places as well then

Reply:
done